examples/src/examples/misc/gizmos.mjs

```diff
@@ -230,18 +231,32 @@ function controls({ observer, ReactPCUI, React, jsx, fragment }) {
  * @param {import('../../options.mjs').ExampleOptions} options - The example options.
  * @returns {Promise<pc.AppBase>} The example application.
  */
-async function example({ canvas, deviceType, data, glslangPath, twgslPath, scriptsPath }) {
+async function example({ pcx, canvas, deviceType, data, glslangPath, twgslPath, scriptsPath }) {
+
     // Class for handling gizmos
     class GizmoHandler {
         _type = 'translate';
 
+        /**
+         * @type {pcx.Gizmo}
+         */
```

Comment:
If there is no other tag like `@private` we keep it single line (to not waste too much vertical space)